mllib/src/test/scala/org/apache/spark/mllib/clustering/PowerIterationClusteringSuite.scala

```diff
@@ -96,14 +96,14 @@ class PowerIterationClusteringSuite extends SparkFunSuite with MLlibTestSparkCon
     }
     val graph = Graph.fromEdges(sc.parallelize(edges, 2), 0.0)
 
-    val model = new PowerIterationClustering()
-      .setK(2)
-      .run(graph)
-    val predictions = Array.fill(2)(mutable.Set.empty[Long])
-    model.assignments.collect().foreach { a =>
-      predictions(a.cluster) += a.id
-    }
-    assert(predictions.toSet == Set((0 to 3).toSet, (4 to 15).toSet))
+//    val model = new PowerIterationClustering()
```

Comment:
Disable this test case because it was blocked by [SPARK-12363](https://issues.apache.org/jira/browse/SPARK-12363).